ios/RCTSensorOrientationChecker.m

```diff
@@ -93,10 +95,11 @@ - (AVCaptureVideoOrientation)convertToAVCaptureVideoOrientation:(UIInterfaceOrie
             return AVCaptureVideoOrientationPortrait;
         case UIInterfaceOrientationPortraitUpsideDown:
             return AVCaptureVideoOrientationPortraitUpsideDown;
+        // Landscape UI and device orientation are opposite
         case UIInterfaceOrientationLandscapeLeft:
-            return AVCaptureVideoOrientationLandscapeLeft;
-        case UIInterfaceOrientationLandscapeRight:
             return AVCaptureVideoOrientationLandscapeRight;
+        case UIInterfaceOrientationLandscapeRight:
+            return AVCaptureVideoOrientationLandscapeLeft;
```

Comment:
While testing orientations, I found that UIInterface and AVCaptureVideo did not match up Landscape values. With *LandscapeLeft = *LandscapeLeft, the saved image was upside down.